File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/objects/RBTV.java
package de.markhaehnel.rbtv.rocketbeanstv.objects;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Getter;

@Getter
public class RBTV {

    @SerializedName("videoId")
    @Expose
    public String videoId;
    @SerializedName("viewerCount")
    @Expose
    public String viewerCount;
    @SerializedName("error")
    @Expose
    public Object error;

}

File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/objects/RBTV.kt
package de.markhaehnel.rbtv.rocketbeanstv.objects

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RBTV {
    @SerializedName("cameras")
    @Expose
    var cameras: List<String> = listOf<String>()

    @SerializedName("viewerCount")
    @Expose
    var viewerCount: String = "0"

    @SerializedName("error")
    @Expose
    var error: Any? = null

}